src/Components/ArticleMainCard.jsx

```diff
@@ -36,6 +41,37 @@ function ArticleMainCard() {
     })
   };
 
+////////////////UPLIFTING FROM THE CHILD COMPONENT/////////////////////////////////
```

Comment:
was there a reason you pulled it out of your comments list component as this component is getting large now

Reply:
Before the api call and the state of commentsList were  in the child component  but now  i thought that i need on the parent child to spread into the list, but I'm sure how fix this stuff so i dropped a msg on slack let me know if is possible have a zoom call with you